docs/introduction/providers.md

```diff
@@ -39,18 +39,17 @@ Here is the list of current providers:
 
 The core Crossplane controller can install provider controllers and CRDs for you
 through its own provider packaging mechanism, which is triggered by the application
-of a `ClusterPackageInstall` resource. For example, in order to request
-installation of the `provider-gcp` package, apply the following resource to the
+of a `Provider` resource. For example, in order to request
+installation of the `provider-aws` package, apply the following resource to the
 cluster where Crossplane is running:
 
 ```yaml
-apiVersion: packages.crossplane.io/v1alpha1
-kind: ClusterPackageInstall
+apiVersion: pkg.crossplane.io/v1alpha1
+kind: Provider
 metadata:
-  name: provider-gcp
-  namespace: crossplane-system
+  name: provider-aws
```

Comment:
any particular reason for updating all of the examples to be AWS?

Reply:
One of them was GCP and other one Azure. I just wanted both to be the same, no particular concerns.